components/PropsTable.tsx

```diff
@@ -0,0 +1,124 @@
+import docgen from '@atom-learning/components/dist/docgen.json'
+import * as React from 'react'
+
+import { styled } from '../stitches.config'
+import { Box, InlineCode, Link, Text } from './'
+
+type PropsTableProps = {
+  for: React.FC
+}
+
+const columns = ['Prop', 'Type', 'Default value', 'Required']
+
+const Table = styled('table', {
+  borderCollapse: 'collapse',
+  width: '100%'
+})
+const Cell = styled('td', {
+  borderBottom: '1px solid $tonal300',
+  color: '$tonal800',
+  fontSize: '$md',
+  fontFamily: '"Inter"',
```

Comment:
Why not use one of the fonts available in `themes`?

Reply:
Yeah we definitely should, or at least update these values in a theme file that we apply globally. I'll probably do that in a different PR where we style things properly, a lot of the styles that are currently set are temporary.